cpp/include/raft/spatial/knn/knn.hpp

```diff
@@ -116,7 +116,7 @@ inline void brute_force_knn(
   std::vector<int> &sizes, int D, float *search_items, int n, int64_t *res_I,
   float *res_D, int k, bool rowMajorIndex = true, bool rowMajorQuery = true,
   std::vector<int64_t> *translations = nullptr,
-  distance::DistanceType metric = distance::DistanceType::L2Unexpanded,
+  distance::DistanceType metric = distance::DistanceType::L2Expanded,
```

Comment:
This here is a potentially breaking change to libraries downstream. I know the expanded form can have quite a signifciant speedup from the exhausted unexpanded form but we'll want to at least verify this doesn't break cuml or euler before changing this. Another option could be to get this PR merged in sooner without this change and then make the change in a different PR once the downstream libraries have been tested.

Reply:
I'll be verifying this fork with cuML PR for verifying these changes in CI